### ISSUE TITLE
New tab page Brave Ads are not confirmed with the server - Only affects 1.18.x

### ISSUE THREAD

**Issue by tmancey** [brave/brave-browser#12818]
## Steps to Reproduce

   1. Join Rewards
   2. View a new tab page ad

## Actual result:

Ad is not confirmed

## Expected result:
Ad should be confirmed

## Reproduces how often: 

Easily reproduced

## Version/Channel Information:

- Can you reproduce this issue with the current release? No
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? No

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? N/A
- Does the issue resolve itself when disabling Brave Rewards? N/A
- Is the issue reproducible on the latest version of Chrome? N/A

**Comment by kjozwiak** [comment]
Closing as https://github.com/brave/brave-core/pull/7217 was merged into `1.18.x`

**Comment by btlechowski** [comment]
Verification passed on

Brave | 1.18.66 Chromium: 87.0.4280.67 (Official Build) dev (64-bit)
-- | --
Revision | 0e5d92df40086cf0050c00f87b11da1b14580930-refs/branch-heads/4280@{#1441}
OS | Windows 7 Service Pack 1 (Build 7601.24544)

Verified test plan from the description
Verified on production.

NTP ad view 
```
[3308:9928:1203/105549.132:VERBOSE3:new_tab_page_ad_event_viewed.cc(78)] Viewed new tab page ad with uuid 17de8096-97e7-4e28-938f-cd2d745a193d and creative instance id 6b563b94-c7d2-4dd8-8e63-8920671e7a99
[3308:9928:1203/105549.133:VERBOSE1:confirmations.cc(91)] Confirming view ad for creative instance id 6b563b94-c7d2-4dd8-8e63-8920671e7a99
[3308:9928:1203/105549.133:VERBOSE1:redeem_unblinded_token.cc(66)] Redeem token
[3308:9928:1203/105549.134:VERBOSE1:redeem_unblinded_token.cc(107)] CreateConfirmation
[3308:9928:1203/105549.134:VERBOSE2:redeem_unblinded_token.cc(108)] POST /v1/confirmation/{confirmation_id}/{credential}
[3308:9928:1203/105549.134:VERBOSE5:redeem_unblinded_token.cc(112)] URL Request:
  URL: https://ads-serve.brave.com/v1/confirmation/52fd5e59-f584-41a4-b175-e3e58b6c5617/eyJwYXlsb2FkIjoie1wiYmxpbmRlZFBheW1lbnRUb2tlblwiOlwiSnFxeHdHcGhIR1V1Z2ZPUGVSK2I4Q2JSNTZBLzBsU0wzZ09BdEtxTVVnWT1cIixcImJ1aWxkQ2hhbm5lbFwiOlwiZGV2XCIsXCJjcmVhdGl2ZUluc3RhbmNlSWRcIjpcIjZiNTYzYjk0LWM3ZDItNGRkOC04ZTYzLTg5MjA2NzFlN2E5OVwiLFwiZXhwZXJpbWVudFwiOnt9LFwicGF5bG9hZFwiOnt9LFwicGxhdGZvcm1cIjpcIndpbmRvd3NcIixcInR5cGVcIjpcInZpZXdcIn0iLCJzaWduYXR1cmUiOiJyblQvVXhpL0RJYjBOQXM5RmFYOWYwWnNwd2Fkc040MDFIdjYzbmplemtmZGVXOHdENFBFMEozUjQva01BQWtKV3Z5TGNwM25XN2R0TmZ3d1VFNkYvdz09IiwidCI6InNIM0ZhakZMcG02dDFxa250Si9Zb21aWEFId0hvOEFtc0RlS1psNWxiejM3MmorSUtTdFduZEFiZHcxeDg2c2RvYkR5dStoc1cwZU9Pa3ZaTlFBYlJ3PT0ifQ==
  Content: {"blindedPaymentToken":"JqqxwGphHGUugfOPeR+b8CbR56A/0lSL3gOAtKqMUgY=","buildChannel":"dev","creativeInstanceId":"6b563b94-c7d2-4dd8-8e63-8920671e7a99","experiment":{},"payload":{},"platform":"windows","type":"view"}
  Content Type: application/json
  Method: BraveAdsUrlRequestMethod::POST
[3308:9928:1203/105549.135:VERBOSE1:refill_unblinded_tokens.cc(67)] No need to refill unblinded tokens as we already have 49 unblinded tokens which is above the minimum threshold of 20
[3308:9928:1203/105549.249:VERBOSE6:new_tab_page_ad_event_viewed.cc(88)] Successfully logged new tab page ad viewed event
[3308:9928:1203/105549.321:VERBOSE1:conversions.cc(92)] URL is not supported for conversions
[3308:9928:1203/105549.321:VERBOSE1:purchase_intent_classifier.cc(92)] Visited URL is not supported for extracting purchase intent
[3308:9928:1203/105549.321:VERBOSE1:page_classifier.cc(65)] Visited URL is not supported for page classification
[3308:9928:1203/105549.413:VERBOSE1:redeem_unblinded_token.cc(125)] OnCreateConfirmation
[3308:9928:1203/105549.413:VERBOSE6:redeem_unblinded_token.cc(127)] URL Response:
  URL: https://ads-serve.brave.com/v1/confirmation/52fd5e59-f584-41a4-b175-e3e58b6c5617/eyJwYXlsb2FkIjoie1wiYmxpbmRlZFBheW1lbnRUb2tlblwiOlwiSnFxeHdHcGhIR1V1Z2ZPUGVSK2I4Q2JSNTZBLzBsU0wzZ09BdEtxTVVnWT1cIixcImJ1aWxkQ2hhbm5lbFwiOlwiZGV2XCIsXCJjcmVhdGl2ZUluc3RhbmNlSWRcIjpcIjZiNTYzYjk0LWM3ZDItNGRkOC04ZTYzLTg5MjA2NzFlN2E5OVwiLFwiZXhwZXJpbWVudFwiOnt9LFwicGF5bG9hZFwiOnt9LFwicGxhdGZvcm1cIjpcIndpbmRvd3NcIixcInR5cGVcIjpcInZpZXdcIn0iLCJzaWduYXR1cmUiOiJyblQvVXhpL0RJYjBOQXM5RmFYOWYwWnNwd2Fkc040MDFIdjYzbmplemtmZGVXOHdENFBFMEozUjQva01BQWtKV3Z5TGNwM25XN2R0TmZ3d1VFNkYvdz09IiwidCI6InNIM0ZhakZMcG02dDFxa250Si9Zb21aWEFId0hvOEFtc0RlS1psNWxiejM3MmorSUtTdFduZEFiZHcxeDg2c2RvYkR5dStoc1cwZU9Pa3ZaTlFBYlJ3PT0ifQ==
  Response Status Code: 201
  Response: {"id":"52fd5e59-f584-41a4-b175-e3e58b6c5617","payload":{},"createdAt":"2020-12-03T09:55:49.951Z","type":"view","modifiedAt":"2020-12-03T09:55:49.951Z","creativeInstanceId":"6b563b94-c7d2-4dd8-8e63-8920671e7a99"}
[3308:9928:1203/105549.413:VERBOSE1:redeem_unblinded_token.cc(147)] FetchPaymentToken
[3308:9928:1203/105549.413:VERBOSE2:redeem_unblinded_token.cc(148)] GET /v1/confirmation/{confirmation_id}/paymentToken
[3308:9928:1203/105549.413:VERBOSE5:redeem_unblinded_token.cc(152)] URL Request:
  URL: https://ads-serve.brave.com/v1/confirmation/52fd5e59-f584-41a4-b175-e3e58b6c5617/paymentToken
  Method: BraveAdsUrlRequestMethod::GET
[3308:9928:1203/105549.645:VERBOSE1:redeem_unblinded_token.cc(163)] OnFetchPaymentToken
[3308:9928:1203/105549.648:VERBOSE6:redeem_unblinded_token.cc(165)] URL Response:
  URL: https://ads-serve.brave.com/v1/confirmation/52fd5e59-f584-41a4-b175-e3e58b6c5617/paymentToken
  Response Status Code: 200
  Response: {"id":"52fd5e59-f584-41a4-b175-e3e58b6c5617","createdAt":"2020-12-03T09:55:49.951Z","type":"view","modifiedAt":"2020-12-03T09:55:49.958Z","creativeInstanceId":"6b563b94-c7d2-4dd8-8e63-8920671e7a99","paymentToken":{"publicKey":"oOJO/xLaCXGW6yhpeIfM4K1X2ln7sgdjTCTCHe8BugE=","batchProof":"Z3kDDoePZDx9bX2tTIY7Jc+JudMxhJA7DMgJa+7GIgUFScpnuH+u2XbrUCp/0/hMXEmIL9P5Mipp5t8XSrLyDA==","signedTokens":["5kkEd0NuBWk7VO8HtBwnkfza8A60RrgnsIhSPadTPGc="]}}
[3308:9928:1203/105549.650:VERBOSE1:redeem_unblinded_token.cc(339)] Added 1 unblinded payment token with an estimated redemption value of 0.01 BAT, you now have 1 unblinded payment tokens
[3308:9928:1203/105549.652:VERBOSE1:ads_impl.cc(512)] Successfully redeemed unblinded token with confirmation id 52fd5e59-f584-41a4-b175-e3e58b6c5617, creative instance id 6b563b94-c7d2-4dd8-8e63-8920671e7a99 and view
```

NTP ad click
```
[3308:9928:1203/105622.152:VERBOSE3:new_tab_page_ad_event_clicked.cc(35)] Clicked new tab page ad with uuid 17de8096-97e7-4e28-938f-cd2d745a193d and creative instance id 6b563b94-c7d2-4dd8-8e63-8920671e7a99
[3308:9928:1203/105622.153:VERBOSE1:confirmations.cc(91)] Confirming click ad for creative instance id 6b563b94-c7d2-4dd8-8e63-8920671e7a99
[3308:9928:1203/105622.153:VERBOSE1:redeem_unblinded_token.cc(66)] Redeem token
[3308:9928:1203/105622.153:VERBOSE1:redeem_unblinded_token.cc(107)] CreateConfirmation
[3308:9928:1203/105622.153:VERBOSE2:redeem_unblinded_token.cc(108)] POST /v1/confirmation/{confirmation_id}/{credential}
[3308:9928:1203/105622.153:VERBOSE5:redeem_unblinded_token.cc(112)] URL Request:
  URL: https://ads-serve.brave.com/v1/confirmation/b471e9e5-1bcf-4a6b-8586-fc6978fae407/eyJwYXlsb2FkIjoie1wiYmxpbmRlZFBheW1lbnRUb2tlblwiOlwiaXBvdDJlMi96THhkUVV1TXZzYktKMDVtMmJqcUtuYlkxSVpadWxyQWNSTT1cIixcImJ1aWxkQ2hhbm5lbFwiOlwiZGV2XCIsXCJjcmVhdGl2ZUluc3RhbmNlSWRcIjpcIjZiNTYzYjk0LWM3ZDItNGRkOC04ZTYzLTg5MjA2NzFlN2E5OVwiLFwiZXhwZXJpbWVudFwiOnt9LFwicGF5bG9hZFwiOnt9LFwicGxhdGZvcm1cIjpcIndpbmRvd3NcIixcInR5cGVcIjpcImNsaWNrXCJ9Iiwic2lnbmF0dXJlIjoibXlRbUNMSTBhb0NRUFh3eVA1ZnovL2NWNEgwMXdOdnNJdk5tcWMzMEFzSUdPdmFXeTBRY0dKYWVFdkRoVEtTc05RNkp1a3ZVcThZWkRxQjlsTW5vcHc9PSIsInQiOiJESVlnZit1REtHQnM2cTM5a3dVazYzOHcxN2ZmakMxTVRzcld4aHo1cWxYZGFiVmlNeTV4RUFZSzJiTXNlUHpYMjB5ck9xcmtwNUlVUXhPWVFMNnZOQT09In0=
  Content: {"blindedPaymentToken":"ipot2e2/zLxdQUuMvsbKJ05m2bjqKnbY1IZZulrAcRM=","buildChannel":"dev","creativeInstanceId":"6b563b94-c7d2-4dd8-8e63-8920671e7a99","experiment":{},"payload":{},"platform":"windows","type":"click"}
  Content Type: application/json
  Method: BraveAdsUrlRequestMethod::POST
[3308:9928:1203/105622.155:VERBOSE1:refill_unblinded_tokens.cc(67)] No need to refill unblinded tokens as we already have 48 unblinded tokens which is above the minimum threshold of 20
[3308:9928:1203/105622.424:VERBOSE1:publisher.cc(515)] Publisher list is empty
[3308:9928:1203/105622.425:VERBOSE6:new_tab_page_ad_event_clicked.cc(47)] Successfully logged new tab page ad clicked event
[3308:9928:1203/105622.447:VERBOSE1:redeem_unblinded_token.cc(125)] OnCreateConfirmation
[3308:9928:1203/105622.447:VERBOSE6:redeem_unblinded_token.cc(127)] URL Response:
  URL: https://ads-serve.brave.com/v1/confirmation/b471e9e5-1bcf-4a6b-8586-fc6978fae407/eyJwYXlsb2FkIjoie1wiYmxpbmRlZFBheW1lbnRUb2tlblwiOlwiaXBvdDJlMi96THhkUVV1TXZzYktKMDVtMmJqcUtuYlkxSVpadWxyQWNSTT1cIixcImJ1aWxkQ2hhbm5lbFwiOlwiZGV2XCIsXCJjcmVhdGl2ZUluc3RhbmNlSWRcIjpcIjZiNTYzYjk0LWM3ZDItNGRkOC04ZTYzLTg5MjA2NzFlN2E5OVwiLFwiZXhwZXJpbWVudFwiOnt9LFwicGF5bG9hZFwiOnt9LFwicGxhdGZvcm1cIjpcIndpbmRvd3NcIixcInR5cGVcIjpcImNsaWNrXCJ9Iiwic2lnbmF0dXJlIjoibXlRbUNMSTBhb0NRUFh3eVA1ZnovL2NWNEgwMXdOdnNJdk5tcWMzMEFzSUdPdmFXeTBRY0dKYWVFdkRoVEtTc05RNkp1a3ZVcThZWkRxQjlsTW5vcHc9PSIsInQiOiJESVlnZit1REtHQnM2cTM5a3dVazYzOHcxN2ZmakMxTVRzcld4aHo1cWxYZGFiVmlNeTV4RUFZSzJiTXNlUHpYMjB5ck9xcmtwNUlVUXhPWVFMNnZOQT09In0=
  Response Status Code: 201
  Response: {"id":"b471e9e5-1bcf-4a6b-8586-fc6978fae407","payload":{},"createdAt":"2020-12-03T09:56:22.960Z","type":"click","modifiedAt":"2020-12-03T09:56:22.960Z","creativeInstanceId":"6b563b94-c7d2-4dd8-8e63-8920671e7a99"}
[3308:9928:1203/105622.447:VERBOSE1:redeem_unblinded_token.cc(147)] FetchPaymentToken
[3308:9928:1203/105622.447:VERBOSE2:redeem_unblinded_token.cc(148)] GET /v1/confirmation/{confirmation_id}/paymentToken
[3308:9928:1203/105622.447:VERBOSE5:redeem_unblinded_token.cc(152)] URL Request:
  URL: https://ads-serve.brave.com/v1/confirmation/b471e9e5-1bcf-4a6b-8586-fc6978fae407/paymentToken
  Method: BraveAdsUrlRequestMethod::GET
[3308:9928:1203/105622.676:VERBOSE1:redeem_unblinded_token.cc(163)] OnFetchPaymentToken
[3308:9928:1203/105622.676:VERBOSE6:redeem_unblinded_token.cc(165)] URL Response:
  URL: https://ads-serve.brave.com/v1/confirmation/b471e9e5-1bcf-4a6b-8586-fc6978fae407/paymentToken
  Response Status Code: 200
  Response: {"id":"b471e9e5-1bcf-4a6b-8586-fc6978fae407","createdAt":"2020-12-03T09:56:22.960Z","type":"click","modifiedAt":"2020-12-03T09:56:22.970Z","creativeInstanceId":"6b563b94-c7d2-4dd8-8e63-8920671e7a99","paymentToken":{"publicKey":"uor3AzFj4OmdCxwetsYD1TxPXZSw40t3j/VOCUyC7Rs=","batchProof":"kfm8CjVJZU8Ae/+GnY7Z0ORxgHwECI/d4L97ZLWJCQSnvGTa8jLNDEY9snf2KJFFKROJg1pGH8YKk9A8xwdKAw==","signedTokens":["YKhgtVVn/F7p4+VbnbmV9I5u2dbaNPPGB7XCKwWwDiY="]}}
[3308:9928:1203/105622.680:VERBOSE1:redeem_unblinded_token.cc(339)] Added 1 unblinded payment token with an estimated redemption value of 0 BAT, you now have 2 unblinded payment tokens
[3308:9928:1203/105622.682:VERBOSE1:ads_impl.cc(512)] Successfully redeemed unblinded token with confirmation id b471e9e5-1bcf-4a6b-8586-fc6978fae407, creative instance id 6b563b94-c7d2-4dd8-8e63-8920671e7a99 and click
```

NTP ad landed
```
[3308:9928:1203/105634.355:VERBOSE1:ad_transfer.cc(118)] Transferred ad for https://try.blockfi.com/credit-card-waitlist/?ref=bravenewtabdec&utm_source=brave&utm_medium=paid&utm_campaign=bravenewtabdec&utm_term=brave_cc&utm_content=12_3
[3308:9928:1203/105634.355:VERBOSE1:confirmations.cc(91)] Confirming landed ad for creative instance id 6b563b94-c7d2-4dd8-8e63-8920671e7a99
[3308:9928:1203/105634.355:VERBOSE1:redeem_unblinded_token.cc(66)] Redeem token
[3308:9928:1203/105634.356:VERBOSE1:redeem_unblinded_token.cc(107)] CreateConfirmation
[3308:9928:1203/105634.356:VERBOSE2:redeem_unblinded_token.cc(108)] POST /v1/confirmation/{confirmation_id}/{credential}
[3308:9928:1203/105634.356:VERBOSE5:redeem_unblinded_token.cc(112)] URL Request:
  URL: https://ads-serve.brave.com/v1/confirmation/ab342fc2-a9a6-4394-8ec4-eeb748d672da/eyJwYXlsb2FkIjoie1wiYmxpbmRlZFBheW1lbnRUb2tlblwiOlwiYUJlMnJYTWdMQU9jaXdSK1JDRjkzeVVCdFlLdXBiVG9qQ2JhUGRyUjZoND1cIixcImJ1aWxkQ2hhbm5lbFwiOlwiZGV2XCIsXCJjcmVhdGl2ZUluc3RhbmNlSWRcIjpcIjZiNTYzYjk0LWM3ZDItNGRkOC04ZTYzLTg5MjA2NzFlN2E5OVwiLFwiZXhwZXJpbWVudFwiOnt9LFwicGF5bG9hZFwiOnt9LFwicGxhdGZvcm1cIjpcIndpbmRvd3NcIixcInR5cGVcIjpcImxhbmRlZFwifSIsInNpZ25hdHVyZSI6IjhRZlhzZng0SGhuRWdNUGYyaU43ck9KdS9nbEg0ck9JNW0vdWpBa2dtV2RsdndGbHhXVzhiMGVxajZjdW9RdlFPUDZLWUhubWt6YnFlVEpwditzODJ3PT0iLCJ0IjoiZ2wwb2plb2NzdDB3MVdyemZ5dHhyL09zbjg4VTJkM2pUVlhkZnpoQldxTDZiYkRNWTUzUkZ6aWR1NnBBUnd1WWxHUWtjMURFcngvMXFFYWo4QjRxbWc9PSJ9
  Content: {"blindedPaymentToken":"aBe2rXMgLAOciwR+RCF93yUBtYKupbTojCbaPdrR6h4=","buildChannel":"dev","creativeInstanceId":"6b563b94-c7d2-4dd8-8e63-8920671e7a99","experiment":{},"payload":{},"platform":"windows","type":"landed"}
  Content Type: application/json
  Method: BraveAdsUrlRequestMethod::POST
[3308:9928:1203/105634.356:VERBOSE1:refill_unblinded_tokens.cc(67)] No need to refill unblinded tokens as we already have 47 unblinded tokens which is above the minimum threshold of 20
[3308:9928:1203/105634.559:VERBOSE6:ad_transfer.cc(128)] Successfully logged transferred ad event
[3308:9928:1203/105634.633:VERBOSE1:redeem_unblinded_token.cc(125)] OnCreateConfirmation
[3308:9928:1203/105634.633:VERBOSE6:redeem_unblinded_token.cc(127)] URL Response:
  URL: https://ads-serve.brave.com/v1/confirmation/ab342fc2-a9a6-4394-8ec4-eeb748d672da/eyJwYXlsb2FkIjoie1wiYmxpbmRlZFBheW1lbnRUb2tlblwiOlwiYUJlMnJYTWdMQU9jaXdSK1JDRjkzeVVCdFlLdXBiVG9qQ2JhUGRyUjZoND1cIixcImJ1aWxkQ2hhbm5lbFwiOlwiZGV2XCIsXCJjcmVhdGl2ZUluc3RhbmNlSWRcIjpcIjZiNTYzYjk0LWM3ZDItNGRkOC04ZTYzLTg5MjA2NzFlN2E5OVwiLFwiZXhwZXJpbWVudFwiOnt9LFwicGF5bG9hZFwiOnt9LFwicGxhdGZvcm1cIjpcIndpbmRvd3NcIixcInR5cGVcIjpcImxhbmRlZFwifSIsInNpZ25hdHVyZSI6IjhRZlhzZng0SGhuRWdNUGYyaU43ck9KdS9nbEg0ck9JNW0vdWpBa2dtV2RsdndGbHhXVzhiMGVxajZjdW9RdlFPUDZLWUhubWt6YnFlVEpwditzODJ3PT0iLCJ0IjoiZ2wwb2plb2NzdDB3MVdyemZ5dHhyL09zbjg4VTJkM2pUVlhkZnpoQldxTDZiYkRNWTUzUkZ6aWR1NnBBUnd1WWxHUWtjMURFcngvMXFFYWo4QjRxbWc9PSJ9
  Response Status Code: 201
  Response: {"id":"ab342fc2-a9a6-4394-8ec4-eeb748d672da","payload":{},"createdAt":"2020-12-03T09:56:35.159Z","type":"landed","modifiedAt":"2020-12-03T09:56:35.159Z","creativeInstanceId":"6b563b94-c7d2-4dd8-8e63-8920671e7a99"}
[3308:9928:1203/105634.633:VERBOSE1:redeem_unblinded_token.cc(147)] FetchPaymentToken
[3308:9928:1203/105634.633:VERBOSE2:redeem_unblinded_token.cc(148)] GET /v1/confirmation/{confirmation_id}/paymentToken
[3308:9928:1203/105634.633:VERBOSE5:redeem_unblinded_token.cc(152)] URL Request:
  URL: https://ads-serve.brave.com/v1/confirmation/ab342fc2-a9a6-4394-8ec4-eeb748d672da/paymentToken
  Method: BraveAdsUrlRequestMethod::GET
[3308:9928:1203/105634.850:VERBOSE1:redeem_unblinded_token.cc(163)] OnFetchPaymentToken
[3308:9928:1203/105634.850:VERBOSE6:redeem_unblinded_token.cc(165)] URL Response:
  URL: https://ads-serve.brave.com/v1/confirmation/ab342fc2-a9a6-4394-8ec4-eeb748d672da/paymentToken
  Response Status Code: 200
  Response: {"id":"ab342fc2-a9a6-4394-8ec4-eeb748d672da","createdAt":"2020-12-03T09:56:35.159Z","type":"landed","modifiedAt":"2020-12-03T09:56:35.172Z","creativeInstanceId":"6b563b94-c7d2-4dd8-8e63-8920671e7a99","paymentToken":{"publicKey":"uor3AzFj4OmdCxwetsYD1TxPXZSw40t3j/VOCUyC7Rs=","batchProof":"Xdb9r1kkxVMIhNoIKGPYOEAaGJiWSnBkLD7kYsxqAAyZQ2/IeYH53jkyedYGi0ZN6JqJzoTuH042R1vwa90gBg==","signedTokens":["FD2l0HNJ2XdkSAO/whc7X6gn9A2yK4yf1ZJ/nrd2gw4="]}}
[3308:9928:1203/105634.852:VERBOSE1:redeem_unblinded_token.cc(339)] Added 1 unblinded payment token with an estimated redemption value of 0 BAT, you now have 3 unblinded payment tokens
[3308:9928:1203/105634.852:VERBOSE1:ads_impl.cc(512)] Successfully redeemed unblinded token with confirmation id ab342fc2-a9a6-4394-8ec4-eeb748d672da, creative instance id 6b563b94-c7d2-4dd8-8e63-8920671e7a99 and landed

```

Verified `Estimated pending rewards` and `Ads received this month` for NTP ad
![image](https://user-images.githubusercontent.com/34715963/100994670-1ffc7e00-3557-11eb-828f-1b67790ef785.png)


Verified NTP ad is shown in ads history
![image](https://user-images.githubusercontent.com/34715963/100994707-27bc2280-3557-11eb-8306-314e3a5b4434.png)
